prompt_windows.go

```diff
@@ -0,0 +1,28 @@
+// +build windows
+
+package prompt
+
+import (
+	"log"
+	"time"
+	"unicode/utf8"
+)
+
+func (p *Prompt) readBuffer(bufCh chan []byte, stopCh chan struct{}) {
+	buf := make([]byte, 1024)
+
+	log.Printf("[INFO] readBuffer start")
+	for {
+		time.Sleep(10 * time.Millisecond)
```

Comment:
Why is the sleep needed here? Is this something windows specific?

Reply:
Nvm I see its in posix as well. So not windows specific. Then y the sleep. Jw.

Reply:
I kept it original code.